src/datasources/products/components/ProductsQueryEditor.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from 'react';
+import { AutoSizeInput, InlineSwitch, MultiSelect, Select, VerticalGroup } from '@grafana/ui';
+import { QueryEditorProps, SelectableValue } from '@grafana/data';
+import { InlineField } from 'core/components/InlineField';
+import { ProductsDataSource } from '../ProductsDataSource';
+import { OrderBy, ProductQuery, Properties } from '../types';
+
+type Props = QueryEditorProps<ProductsDataSource, ProductQuery>;
+
+export function ProductsQueryEditor({ query, onChange, onRunQuery, datasource }: Props) {
+  query = datasource.prepareQuery(query);
+
+  const handleQueryChange = useCallback((query: ProductQuery, runQuery = true): void => {
+    onChange(query);
+    if (runQuery) {
+      onRunQuery();
+    }
+  }, [onChange, onRunQuery]);
+
+  const onPropertiesChange = (items: Array<SelectableValue<string>>) => {
+    if (items !== undefined) {
+      handleQueryChange({ ...query, properties: items.map(i => i.value as Properties) });
+    }
+  };
+
+  const onOrderByChange = (item: SelectableValue<string>) => {
+    handleQueryChange({ ...query, orderBy: item.value });
+  }
+
+  const onDescendingChange = (isDescendingChecked: boolean) => {
+    handleQueryChange({ ...query, descending: isDescendingChecked });
+
+  }
+
+  const recordCountChange = (event: React.FormEvent<HTMLInputElement>) => {
+    const value = parseInt((event.target as HTMLInputElement).value, 10);
+    handleQueryChange({ ...query, recordCount: value });
+  }
+
+  return (
+    <>
+      <VerticalGroup>
+        <InlineField label="Properties" labelWidth={18} tooltip={tooltips.properties}>
+          <MultiSelect
+            placeholder='Select properties to fetch'
```

Comment:
```suggestion
            placeholder="Select properties to fetch"
```

Typically, In JSX/TSX, we use double quotes for representing strings.

Reply:
Update in all places

Reply:
updated to doudle quotes